pulsar/client_impl_test.go

```diff
@@ -222,3 +221,84 @@ func TestTopicPartitions(t *testing.T) {
 	assert.Equal(t, len(partitions), 1)
 	assert.Equal(t, partitions[0], topic)
 }
+
+func TestNamespaceTopicsNamespaceDoesNotExit(t *testing.T) {
+	c, err := NewClient(ClientOptions{
+		URL: serviceURL,
+	})
+	if err != nil {
+		t.Errorf("failed to create client error: %+v", err)
+		return
+	}
+	defer c.Close()
+	ci := c.(*client)
+
+	// fetch from namespace that does not exist
+	name := generateRandomName()
+	topics, err := ci.namespaceTopics(fmt.Sprintf("%s/%s", name, name))
+	assert.Equal(t, 0, len(topics))
+}
+
+func TestNamespaceTopics(t *testing.T) {
+	name := generateRandomName()
+	namespace := fmt.Sprintf("public/%s", name)
+	namespaceUrl := fmt.Sprintf("http://localhost:8080/admin/v2/namespaces/%s", namespace)
```

Comment:
In `test_helper.go` we have: 

```
	serviceURL    = "pulsar://localhost:6650"
	serviceURLTLS = "pulsar+ssl://localhost:6651
```

we could add the `http://localhost:8080` there as well